### DISCUSSION TITLE
about ethers@^6

### DISCUSSION THREAD

**Discussion by zx61** [WalletConnect/walletconnect-monorepo#2292] [Developer Support] [answered]
ethers@^6 has significant advantages in speed..Is there a plan for ethers@^6?

**Comment by xzilja** [comment, marked as answer]
In terms of wagmi, they will be switching to https://viem.sh instead of ethers very soon, which has even more advantages over ethers v6. We might follow up with new non wagmi client for web3modal based on ethers as well.
